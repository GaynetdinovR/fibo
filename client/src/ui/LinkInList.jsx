import styles from "../styles/Ui.module.sass";
import classNames from "classnames";
import LinkUI from "./LinkUI.jsx";

const LinkInList = ({ children, liClass, aClass, href }) => {
	return (
		<li className={classNames(liClass, styles.link_in_list__item)}>
			<LinkUI className={aClass} href={href}>
				{children}
			</LinkUI>
		</li>
	);
};

export default LinkInList;