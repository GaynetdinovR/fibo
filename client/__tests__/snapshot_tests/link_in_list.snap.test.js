import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import LinkInList from "../../src/ui/LinkInList.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<LinkInList>LinkInList</LinkInList>).toJSON();
	expect(tree).toMatchSnapshot();
});
