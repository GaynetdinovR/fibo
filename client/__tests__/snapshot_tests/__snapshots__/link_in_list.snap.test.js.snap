// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="link_in_list__item"
>
  <a
    className="link"
  >
    LinkInList
  </a>
</li>
`;
